channel/consensus_channel/consensus_channel.go

```diff
@@ -359,7 +378,12 @@ func (p *Proposal) equal(q *Proposal) bool {
 
 // isAddProposal returns true if the proposal contains a non-nil toAdd and a nil toRemove.
```

Comment:
These comments are slightly misleading: `ToAdd == Add{}` does not mean that `ToAdd` is `nil`!

Reply:
This ended up getting removed in https://github.com/statechannels/go-nitro/pull/483/commits/133f10fc803bc94c4f7bc964f7e05fd0bb5a9a9f